tika-parsers/src/test/java/org/apache/tika/parser/recognition/ObjectRecognitionParserTest.java

```diff
@@ -68,22 +78,49 @@ public void jpegTesorflowTest() throws IOException, TikaException, SAXException
 
     @Ignore("Configure Rest API service")
     @Test
-    public void testREST() throws Exception {
-        try (InputStream stream = loader.getResourceAsStream(CONFIG_REST_FILE)){
+    public void jpegRESTObjRecTest() throws Exception {
+        String[] expectedObjects = {"Egyptian cat", "tabby, tabby cat"};
+        doRecognize(CONFIG_REST_FILE_OBJ_REC, CAT_IMAGE_JPEG,
+                ObjectRecognitionParser.MD_KEY_OBJ_REC, expectedObjects);
+
+
+    }
+
+    @Ignore("Configure Rest API service")
+    @Test
+    public void jpegRESTim2txtTest() throws Exception {
+        String[] expectedObjects = {"baseball", "bat", "field"};
+        doRecognize(CONFIG_REST_FILE_IM2TXT, BASEBALL_IMAGE_JPEG,
+                ObjectRecognitionParser.MD_KEY_IMG_CAP, expectedObjects);
+
+    }
+
+    @Ignore("Configure Rest API service")
+    @Test
+    public void pngRESTim2txtTest() throws Exception {
+        String[] expectedObjects = {"Egyptian cat", "tabby, tabby cat"};
+        doRecognize(CONFIG_REST_FILE_IM2TXT, BASEBALL_IMAGE_PNG,
```

Comment:
I know it's hard coded only to work with jpeg for now..I'm thinking of a better way to pass the image format into the comp' graph.